### PR TITLE
Added MTU request method

### PR DESCRIPTION
**Pull request by valexi7**
I added a method to request MTU change discussed in issue:
MTU size request #25

Usage:
`connection?.requestMTU(512)`

- Fixed bug on function "onReadRemoteRssi", rsii value got newer updated, because it was pointing to itself
- Added method for reading remote rssi.

Usage:
`connection?.readRSSI()`

### CODE REVIEW

**Comment by LeandroSQ** [comment]
Hello, thank you for your PR :)
I appreciate it, sorry for the delay, life has been crazy.

Is it possible to remove the files not associated with the change from this PR? e.g .idea folder, or the gradle files.

File: .idea/compiler.xml
Comment (LeandroSQ) [inline .idea/compiler.xml:1]: Is it possible to remove this file from this PR?

File: .idea/misc.xml
Comment (LeandroSQ) [inline .idea/misc.xml:1]: Is it possible to remove this file from this PR?

File: build.gradle
Comment (LeandroSQ) [inline build.gradle:1]: Is it possible to remove this file from this PR?

File: gradle/wrapper/gradle-wrapper.properties
Comment (LeandroSQ) [inline gradle/wrapper/gradle-wrapper.properties:1]: Is it possible to remove this file from this PR?

File: lib/build.gradle
Comment (LeandroSQ) [inline lib/build.gradle:1]: Is it possible to remove this file from this PR?

**Review by LeandroSQ — CHANGES_REQUESTED** [review]
Awesome work, thank you :)

The only nitpicking is the project-level files.

**Comment by karansingla-dev** [comment]
@LeandroSQ  when will you merge this branch? I think when I get large-size data from the BLE device then it disconnected from the Android device.

**Comment by valexi7** [comment]
> @LeandroSQ when will you merge this branch? I think when I get large-size data from the BLE device then it disconnected from the Android device.

Hi,
In the meantime you can use this:
https://jitpack.io/#valexi7/android-ble-made-easy

**Review by LeandroSQ — APPROVED** [review]

**Comment by karansingla-dev** [comment]
@valexi7 
this is I am getting

image

**Comment by LeandroSQ** [comment]
Some files were deleted on this PR, I reverted the deletion and the most recent on this feature is [feature/mtu](https://github.com/LeandroSQ/android-ble-made-easy/tree/feature/mtu) branch. Will create another PR so we can put this into master and plan a new release.

Thank you @valexi7 for this contribution, I appreciate it.

**Comment by karansingla-dev** [comment]
@LeandroSQ when are you planning to merge this branch into the main?

**Comment by LeandroSQ** [comment]
Yes, tracking on #26

**Comment by valexi7** [comment]
> @valexi7 this is I am getting
> 
> image

If you have Jitpack implementation it should work like this:

`implementation 'com.github.valexi7:android-ble-made-easy:-SNAPSHOT'`

**Comment by karansingla-dev** [comment]
@valexi7  See this
image

**Pull request merged**